<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportSetDialog</class>
 <widget class="QDialog" name="ImportSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import set</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Import/Create set</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="chooseFileButton">
     <property name="text">
      <string>Choose file to import from</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nameLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="setInput">
     <property name="placeholderText">
      <string>Enter terms</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="delimLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Delimeter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="delimEdit">
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="termFirstButton">
     <property name="text">
      <string>Term first</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="defFirstButton">
     <property name="text">
      <string>Definition first</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="previewButton">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="importButton">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
